aten/src/ATen/native/quantized/cpu/qconcat.cpp

```diff
@@ -22,7 +22,7 @@ bool is_valid_quantization_scheme(Tensor t) {
 template <bool ReLUFused>
 Tensor quantized_cat(
     const std::vector<Tensor>& qxs,
-    int64_t axis,
+    int64_t dim,
```

Comment:
it's probably fine now, but in general - don't mix unrelated changes in the same PR

Reply:
My bad -- this was bothering me a lot, and I was too lazy to have a stack :) Will not do that again!